src/components/Tabs.js

```diff
@@ -58,6 +58,7 @@ class Tabs extends Component {
                     handleSelect,
                     isActive: child.props.to === selectedTab,
                     activeStyle: activeLinkStyle,
+                    disableInlineStyles: child.props.disableInlineStyles !== undefined ? child.props.disableInlineStyles : disableInlineStyles,
```

Comment:
Haha ok. I don't know actually, thought it would be nice to have that option, but you're right... not sure if there's a use case XD. Will fix.